test/indexer/__init__.py

```diff
@@ -100,7 +101,10 @@ def _load_canned_bundle(cls, bundle: SourcedBundleFQID) -> Bundle:
 
 class IndexerTestCase(ElasticsearchTestCase, CannedBundleTestCase):
     index_service: IndexService
-    source = DSSSourceRef.for_dss_endpoint('test')
+    # We manually change `id` and `name` for the test so they are easy to
+    # distinguish in a document.
+    source = DSSSourceRef(id='test_source_id',
```

Comment:
This is inconsistent with azul.plugins.repository.dss.Plugin.lookup_source_id. I'm surprised this doesn't already cause problems. I understand why we want to make sure those two aren't accidentally swapped. Let's change `lookup_source_id` and `for_dss_endpoint` to UUID v5 hash the name and use the hash as the id.